ocfweb/main/staff_hours.py

```diff
@@ -9,6 +13,8 @@ def staff_hours(request):
         {
             'title': 'Staff Hours',
             'staff_hours': get_staff_hours(),
+            'today': time.strftime('%A'),
```

Comment:
oh, this is for staff_hours, not lab_hours.

(lab_hours I recently renamed it to `weekday` which is why I was thinking it'd be consistent)

Seems fine as-is in that case.